website/docs/reference/appsmith-framework/widget-actions/post-message.md

```diff
@@ -55,7 +55,7 @@ windowMessageListener(
 
 | Argument | Description |
 | --- | --- |
-| domain | This is the address of the website that sends the message (`https://mywebsite.com`). The app only listens to messages from the given domain when embedded. If the app is embedded in some other website(`https://myother-website.com`)the callback  won’t be triggered. If an active action is already in place, it won't be overridden and a warning appears in the console. |
+| domain | This is the address of the website that sends the message (`https://mywebsite.com`). The app only listens to messages from the given domain when embedded. If the app is embedded in some other website(`https://myother-website.com`)the callback  won’t be triggered. If an active action is already in place, it won't be overridden and a warning appears in the console. You cannot use ** * ** as an input for the domain because listening to all websites is not allowed.|
```

Comment:
I think - this will be better - You cannot use ** * ** in the domain, as Appsmith supports listening to specific websites. @eco-monk - Is this technically correct?

Reply:
Yep, we allow listening to specific websites only.